websites/P/Peario/dist/metadata.json

```
@@ -0,0 +1,44 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"name": "Dark_Ville",
+		"id": "638080361179512853"
+	},
+	"service": "Peario",
+	"description": {
+		"nl": "Deel je streams. Peario zorgt er voor dat je Stremio streams kan delen met je vrienden.",
+		"en": "Share your streams. Peario allows you to watch Stremio streams with your friends."
+	},
+	"url": [
+		"peario.xyz",
+		"www.peario.xyz"
+	],
```

www.peario.xyz doesn't exist
```suggestion
	"url": "peario.xyz",
```